app/api/helpers/errors.py

```diff
@@ -78,3 +79,16 @@ class BadRequestError(ErrorResponse):
     """
     status = 400
     title = 'Bad Request'
+
+
+class RequestLimitError(ErrorResponse):
+    """
+    Default class for 429 Error
+    """
+    status = 429
+    title = 'Request Limit Exceeded'
+
+
+@error_blueprint.app_errorhandler(429)
+def handle_429(error):
+    raise RequestLimitError({'source': ''}, error.description).respond()
```

Comment:
Why this? Keep it consistent with other handlers

Reply:
Can you please tell me where other error handlers are implemented?

Reply:
They are present in __init__.py

Reply:
Ok @iamareebjamal. Working on it. Thanks for reviewing!